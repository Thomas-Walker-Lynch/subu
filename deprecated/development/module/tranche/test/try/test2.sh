#!/bin/bash -x
./tranche test2.trc.c >test2stdout.dat
diff -q test2.c test2.c.expected
diff -q test2.h test2.h.expected
diff -q test2stdout.dat test2stdout.dat.expected
rm test2.c test2.h test2stdout.dat


