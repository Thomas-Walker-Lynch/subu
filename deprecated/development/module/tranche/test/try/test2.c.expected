

#include "test2.h"


int f(int x){
  return x;
}


